charts/newrelic-logging/tests/fluentbit_pod_label_test.yaml

```diff
@@ -0,0 +1,43 @@
+suite: test fluent-bit pods labels
+templates:
+  - templates/daemonset.yaml
+  - templates/configmap.yaml
+  - templates/persistentvolume.yaml
+release:
+  name: my-release
+  namespace: my-namespace
+tests:
+- it: pod labels are correct
```

Comment:
I would rename this to `multiple pod labels are set properly` and create a new one for `a single pod label is set properly`.